### PR TITLE
CS Fixes

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Review by ogizanagi — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by chalasr** [comment]
travis random failure unrelated

File: src/Symfony/Component/Console/Application.php [hunk 1]
```diff
@@ -197,7 +197,6 @@ public function doRun(InputInterface $input, OutputInterface $output)
             $e = $this->runningCommand = null;
             // the command name MUST be the first element of the input
             $command = $this->find($name);
-        } catch (\Exception $e) {
         } catch (\Throwable $e) {
         }
         if (null !== $e) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Application.php]: then this if is useless (and the initialization of $e also)

File: src/Symfony/Component/Console/Application.php [hunk 2]
```diff
@@ -813,7 +812,6 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
             } else {
                 $exitCode = ConsoleCommandEvent::RETURN_CODE_DISABLED;
             }
-        } catch (\Exception $e) {
         } catch (\Throwable $e) {
         }
         if (null !== $e) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Application.php]: same (init + if can be removed)

File: src/Symfony/Component/Yaml/Parser.php [hunk 3]
```diff
@@ -103,7 +103,6 @@ public function parse($value, $flags = 0)
 
         try {
             $data = $this->doParse($value, $flags);
-        } catch (\Exception $e) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Yaml/Parser.php]: same

File: src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php [hunk 4]
```diff
@@ -88,7 +52,7 @@ private function getType(\ReflectionParameter $parameter)
         if (!$type = $parameter->getType()) {
             return;
         }
-        $typeName = $type instanceof \ReflectionNamedType ? $type->getName() : $type->__toString();
```
Comment (chalasr) [inline src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php]: ReflectionType::__toString() is deprecated

Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php]: right - and there are more of it in the code base, isn't it?

Comment (chalasr) [inline src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php]: indeed, all fixed. addressed your other comments also

File: src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php [hunk 5]
```diff
@@ -51,7 +51,6 @@ public static function castClosure(\Closure $c, array $a, Stub $stub, $isNested,
                 unset($v->value['position'], $v->value['isVariadic'], $v->value['byReference'], $v);
             }
         }
-        ;
```
Comment (nicolas-grekas) [inline src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php]: not sure about this one

Comment (chalasr) [inline src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php]: me either, applied fabbot patch blindly. reverted

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**